Osiris/Hacks/Aimbot.cpp

```diff
@@ -97,6 +97,20 @@ static bool canScan(Entity* entity, const Vector& destination, const WeaponInfo*
     return false;
 }
 
+static void setRandomSeed(int seed) noexcept
+{
+    using randomSeedFn = void(*)(int);
+    static auto randomSeed{ reinterpret_cast<randomSeedFn>(GetProcAddress(GetModuleHandleA("vstdlib.dll"), "RandomSeed")) };
+    randomSeed(seed);
+}
+
+static float getRandom(float min, float max) noexcept
```

Comment:
get rid of setRandomSeed and replace this with the code below, that way random rcs will work
```
float getRandom(float bottom, float top) noexcept
{   
    std::random_device rd{};
    std::mt19937 generator(rd());
    std::uniform_real_distribution<float> dis(bottom, top);
    return dis(generator);
}
```

Reply:
thanks I will check it :)

Reply:
Can you explain how to do this, what needs to be removed? when I add your code its gives me 100 error, admittedly I dont know what im doing lol.
![Capture](https://user-images.githubusercontent.com/75226968/120355814-ec32c300-c303-11eb-8fb7-83be4f079e55.PNG)
![Capture2](https://user-images.githubusercontent.com/75226968/120355823-ee951d00-c303-11eb-8939-008de0604b0a.PNG)

Reply:
you are missing #include <random>

Reply:
I put that in aimbot settings?

Reply:
And what do I do with setRandomseed, do I delete the set of lines for that code or just rename

Reply:
> 
> 
> **get rid of setRandomSeed** and replace this with the code below, that way random rcs will work
> 
> ```
> float getRandom(float bottom, float top) noexcept
> {   
>     std::random_device rd{};
>     std::mt19937 generator(rd());
>     std::uniform_real_distribution<float> dis(bottom, top);
>     return dis(generator);
> }
> ```